packages/cfd/src/Components/props.types.ts

```diff
@@ -276,6 +276,12 @@ export type TJurisdictionData = {
 
 export type TDetailsOfEachMT5Loginid = DetailsOfEachMT5Loginid & {
     display_login?: string;
+    white_label_links?: {
+        webtrader_url: string;
+        android: string;
+        ios: string;
+        windows: string;
+    };
```

Comment:
just checking why not we add this type to `packages/cfd/src/types/types.ts` so thaty we can reuse it in `packages/cfd/src/features/Containers/mt5-trade-modal.tsx`

Reply:
putting it in `prop.types` because `DetailsOfEachMT5Loginid` is used repetitively there 

plus`/features` directory can be deprecated. Nobody is working on it anymore, we have migrated to `packages/tradershub/src/features/cfd` for all v2 related stuff